### PR TITLE
UCL-262 Cleanup non-quantum color variables

### PR DESCRIPTION
**Pull request by MichalGorskiOnegini**

### CODE REVIEW

File: src/Form/FileUpload/FileUpload.module.scss [hunk 1]
```diff
@@ -29,11 +29,10 @@
   @include transition(all, 0.2s, ease-in-out);
   &.drag-active {
     @include outline(
-      var(--color-primary),
+      var(--color-primary300),
```
Comment (pawel-napieracz) [inline src/Form/FileUpload/FileUpload.module.scss:32]: border: bluegray-500,
background: bluegray-25

File: src/Form/FileUpload/FileItem/FileItem.modules.scss [hunk 2]
```diff
@@ -26,10 +26,6 @@
     font-size: 1.25rem;
   }
 
-  .progress-bar {
-    background-color: var(--light-pink);
```
Comment (pawel-napieracz) [inline src/Form/FileUpload/FileItem/FileItem.modules.scss:30]: maybe it should be: black-10?

Comment (pawel-napieracz) [reply src/Form/FileUpload/FileItem/FileItem.modules.scss:30]: https://www.figma.com/file/zAzyG3fmBFKrdrVzFeEYAv/Quantum---2.-%F0%9F%8C%9E-Light-Components?node-id=12076-286905&t=JVKQYe5hvfU0OeJQ-0

File: stories/BaseStyling/BaseStyling.stories.mdx [hunk 3]
```diff
@@ -127,10 +127,6 @@ Will get translated to the CSS variable `--input-border-style`. This is the `bor
 
 Will get translated to the CSS variable `--input-background-color`. This is the `background-color` for every [Input component](/story/components-inputs-input-wrapper--input-wrapper).
 
-## dragBackgroundColor
```
Comment (pawel-napieracz) [inline stories/BaseStyling/BaseStyling.stories.mdx:130]: couldn't do it in line 51, but would be nice to sync that list.
Also we have these colors in our plugin. Please update our properties there as well (see comment in `BaseStyling.tsx:18`)

Comment (pawel-napieracz) [reply stories/BaseStyling/BaseStyling.stories.mdx:130]: And don't remove that paraghaph plx :D

**Review by pawel-napieracz — CHANGES_REQUESTED** [review]

File: src/Form/FileUpload/FileItem/FileItem.modules.scss (hunk 2 above)
Comment (MichalGorskiOnegini) [reply src/Form/FileUpload/FileItem/FileItem.modules.scss:30]: ok

File: src/_BaseStyling_/BaseStyling.tsx [hunk 4]
```diff
@@ -148,8 +149,8 @@ export const BaseStyling = ({ children, properties = {} }: Props) => {
     inputBorderWidthFocus: "2px",
     inputBorderStyle: "solid",
     inputBackgroundColor: "var(--light)",
-    dragBackgroundColor: "#1313191A",
-    dragBorderStyle: "dashed",
+    dragBackgroundColor: "var(--color-blue-grey25-transparent)",
```
Comment (pawel-napieracz) [inline src/_BaseStyling_/BaseStyling.tsx:152]: you don't use that color :(

File: stories/BaseStyling/BaseStyling.stories.mdx [hunk 5]
```diff
@@ -281,10 +281,6 @@ Will get translated to the CSS variable `--warning`. Used in the [StatusIndicato
 
 Will get translated to the CSS variable `--light`. This color is used in all components where the color `white` is used by default. Whenever dark-mode is introduced, you can easily make a dark color scheme.
 
-## grey
```
Comment (pawel-napieracz) [inline stories/BaseStyling/BaseStyling.stories.mdx:284]: please add in docs missing colors + sync /.storybook/addon/src/components/PanelContent.tsx

**Review by pawel-napieracz — CHANGES_REQUESTED** [review]

**Review by pawel-napieracz — APPROVED** [review]

**Pull request merged**